src/pages/docs/creators/tools/yagna/requestor-uninstallation.md

```diff
@@ -39,7 +39,7 @@ Yagna is manually installed by extracting from an archive. Simply locate and del
 If they are in your system's PATH, you can locate them with the following command:
 
 ```sh
-whereis yagna gftp
+where yagna gftp
```

Comment:
is this indeed the correct command on Windows?

Reply:
yes,